src/utils/encoding.ts

```diff
@@ -283,7 +283,7 @@ export function parseRedeemRequestStatus(
     // now check if pending is actually expired
     const maxPeriod = Math.max(req.period.toNumber(), redeemPeriod);
     const openTime = req.opentime.toNumber();
-    if (openTime + maxPeriod > activeBlockCount) {
+    if (openTime + maxPeriod <= activeBlockCount) {
```

Comment:
Shouldn't this be unequal comparison - only `<` not `<=`? Because I think that in case when current block is the same as the expiration block, the request is still valid and becomes expired only in the block after:
https://github.com/interlay/interbtc/blob/c88fc96634160ed585fb8467a98992f52b622a69/crates/security/src/lib.rs#L

Please correct me if I'm wrong.

Reply:
Yes, you are right. Good point. Thanks!